oodeel/methods/base.py

```diff
@@ -138,20 +135,29 @@ def _load_feature_extractor(
         self,
         model: Callable,
         feature_layers_id: List[Union[int, str]] = None,
+        input_layer_id: Optional[Union[int, str]] = None,
     ) -> Callable:
         """
         Loads feature extractor
 
         Args:
             model: a model (Keras or PyTorch) to load.
+            input_layer_id (List[int]): = list of str or int that identify the input
+                layer of the feature extractor.
+                If int, the rank of the layer in the layer list
+                If str, the name of the layer. Defaults to None.
+            feature_layers_id (List[int]): list of str or int that identify
+                features to output.
+                If int, the rank of the layer in the layer list
+                If str, the name of the layer. Defaults to [-1],
```

Comment:
The order of the docstring is not the same as the arguments: `feature_layers_id` is before `input_layer_id` in the signature.